services/xcode-select-service.ts

```diff
@@ -0,0 +1,23 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import {join} from "path";
+
+export class XcodeSelectService implements IXcodeSelectService {
```

Comment:
I would love to see tests for this service that mock childProcess and it's output